File: android/apolloui/src/main/java/io/muun/apollo/presentation/ui/fragments/recommended_fee/RecommendedFeeView.java
package io.muun.apollo.presentation.ui.fragments.recommended_fee;

import io.muun.apollo.domain.model.CurrencyDisplayMode;
import io.muun.apollo.domain.model.PaymentContext;
import io.muun.apollo.domain.model.PaymentRequest;
import io.muun.apollo.presentation.ui.base.BaseView;

interface RecommendedFeeView extends BaseView {

    void setCurrencyDisplayMode(CurrencyDisplayMode currencyDisplayMode);

    void setPaymentContext(PaymentContext paymentContext, PaymentRequest paymentRequest);
}

File: android/apolloui/src/main/java/io/muun/apollo/presentation/ui/fragments/recommended_fee/RecommendedFeeView.kt
package io.muun.apollo.presentation.ui.fragments.recommended_fee

import io.muun.apollo.domain.model.BitcoinUnit
import io.muun.apollo.presentation.ui.base.BaseView
import newop.EditFeeState

interface RecommendedFeeView : BaseView {

    fun setBitcoinUnit(bitcoinUnit: BitcoinUnit)

    fun setState(state: EditFeeState)
}